[Datapackage][m]: Remove sample and schema from top level data package

risenW opened the pull request:
This pull fixes #64

sglavoie commented on src/components/Upload/index.jsx:
```diff
@@ -114,83 +121,91 @@ class Upload extends React.Component {
             validFile: false,
-            errorMsg: "An error occured when trying to load the file!", file: {}
-          })
+            errorMsg: "An error occured when trying to load the file!",
```
Fixing typo:
```suggestion
            errorMsg: "An error occurred when trying to load the file!",
```

sglavoie reviewed (commented):
Thanks @risenW! :slightly_smiling_face: I wasn't sure about this line: https://github.com/datopian/gift-publisher/pull/65/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R122

Apart from that, I just fixed a few typos.

risenW commented on src/App.js:
```diff
@@ -131,27 +117,35 @@ export class DatasetEditor extends React.Component {
-      dataset.model = fdp.model;
+    let model;
+    let schema;
+    if (dataset.resources.length > 1) {
```
It's a subtle one, but it's correct. An uploaded resource gets added to the data package before this test takes place. So at any given time, before we get here, a resource already exist but it may not have a schema and model

sglavoie replied:
Oh great, thanks for taking the time to shoot me an explanation, that makes sense. :+1:

risenW commented:
> Thanks @risenW! 🙂 I wasn't sure about this line: https://github.com/datopian/gift-publisher/pull/65/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R122
> 
> Apart from that, I just fixed a few typos.

I dropped a comment. Thanks

sglavoie commented:
@risenW thank you, I saw it and resolved the conversation since there was nothing left to do with my previous comment. LGTM. :slightly_smiling_face: :+1:
